Hoxtygen/phone_bookr
# Access phone contacts.

##  What does this PR do
Provided access to phone contacts

##  Description of task to be completed
* Install permission_handler package
* Install contacts_service package
* Add necessary info to android `androidManifest.xml`  and iOs `info.plist` files respectively
* Create the widget for seeking permission to access contact
* Create widget for displaying contact when permission is given
* Create dialog for displaying message when permission is denied

##  How should this manually be tested
* Clone the repository
* Open a terminal and run `git clone `repo_url`
* Cd into the project directory and open in a text editor of choice; 
* Run the application.
* Accept permission request, and  phone contacts should be showing in the app

##  Screenshots (If appropriate)
![Screenshot_1616670545](https://user-images.githubusercontent.com/19408669/112463881-13e53800-8d63-11eb-819f-840b161d7d44.png)


##  Questions (If any):
